packages/rocketchat-ui-account/account/accountProfile.html

```diff
@@ -59,6 +59,9 @@
 				<div class="submit">
 					<button class="button"><i class="icon-send"></i><span>{{_ "Save_changes"}}</span></button>
 				</div>
+				<div class="logoutOthers">
+					<button class="button">Logout Others</button>	
```

Comment:
Ok, I can do that and raise a PR. However, now my `development` branch is out of sync with the upstream repo. 

How do I pull in changes and update both my forked repo and cloned repo; without losing my changes? I have a feeling I might need to `git rebase` and / or `git stash`; though I am not exactly sure about the steps.

Also, what about Strings in other languages? Do I also have to provide those? Or would we leave that as another issue, for future work, to be merged from another PR?